blazor/datagrid/cell-edit-types.md

```diff
@@ -984,6 +984,71 @@ The `SfTextBox` component is defined inside the EditTemplate and changes can be
 }
 ```
 
+### Prevent enter key functionality in multiline textbox while editing
+
+By default, pressing the ENTER key will save the changes in the Grid when focus is in the Grid Edit form (both normal and dialog edit mode). In Multiline Textbox, the ENTER key should add a new line break instead of saving the edited changes while editing. This can be achieved by using Microsoft JsInterop.
+
+In the following sample, the Multiline Textbox is rendered in the Customer ID column. The Microsoft JsInterop is used to call a JavaScript function in the SfTextBox's OnFocus event. Here, the ENTER key action can be prevented from occurring by using the `stopPropagation()` method.
```

Comment:
In the following sample, Multiline Textbox is rendered in the Customer ID column. The Microsoft JsInterop is used to call a JavaScript function in the SfTextBox's OnFocus event. Here, the ENTER key action can be prevented from occurring by using the `stopPropagation()` method.